modules/gdnative/pluginscript/pluginscript_instance.cpp

```diff
@@ -0,0 +1,151 @@
+// Godot imports
```

Comment:
Don't forget to add the Godot copyright header to all .cpp and .h files.

Reply:
done.